v5_MIGRATION_GUIDE.md

```diff
@@ -19,5 +20,46 @@ Now, you can tokenize with just the card details:
 
 // TODO: code snippet of card integration in v5
 
+## Venmo
+
+The Venmo integration has been updated to allow for more flexibility with app switching and 
+activity result handling. 
+
+`VenmoLauncher` has been added to handle the app switching portion of the Venmo flow for user 
+authentication via the Venmo app. This class uses the Android Activity Result API and therefore 
+must be instantiated in the `OnCreate` method of your Activity or Fragment.
+
+`BraintreeClient` and `VenmoClient` no longer require references to Fragment or Activity and do not 
+need to be instantiated in `OnCreate`.
+
+```kotlin
 
+class MyActivity : FragmentActivity() {
+    
+    private lateinit var venmoLauncher: VenmoLauncher
+    private lateinit var braintreeClient: BraintreeClient
+    private lateinit var venmoClient: VenmoClient
+    
+    @override fun onCreate(savedInstanceState: Bundle?) {
+        venmoLauncher = VenmoLauncher(this) { authChallengeResult ->
+            venmoClient.tokenizeVenmoAccount(authChallengeResult) { venmoAccountNonce, error ->
+                error?.let { /* handle error */ }
+                venmoAccountNonce?.let { /* handle Venmo account nonce */ }
+            }
+        }
+    }
+    
+    fun initializeClients() {
+        braintreClient = BraintreeClient(context, "TOKENIZATION_KEY_OR_CLIENT_TOKEN")
+        venmoClient = VenmoClient(braintreeClient)
+    }
+    
+    fun launchVenmoFlow() {
```

Comment:
Do we need this func? Looks like it isn't called anywhere in this sample snippet

Reply:
This is what would be invoked on the click of the pay with venmo button or however they are starting the payment flow. I didn't include the button/setup logic in this example - would that make it more clear or a code comment?

Reply:
Updated method name 👍